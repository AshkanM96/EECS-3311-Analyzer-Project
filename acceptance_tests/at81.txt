union
open_set_enumeration
integer_constant(1)
integer_constant(2)
integer_constant(3)
close_set_enumeration
open_set_enumeration
integer_constant(3)
integer_constant(4)
integer_constant(5)
close_set_enumeration
type_check
evaluate
reset
intersect
open_set_enumeration
integer_constant(1)
integer_constant(2)
integer_constant(3)
close_set_enumeration
open_set_enumeration
integer_constant(3)
integer_constant(4)
integer_constant(5)
close_set_enumeration
type_check
evaluate
reset
difference
open_set_enumeration
integer_constant(1)
integer_constant(2)
integer_constant(3)
close_set_enumeration
open_set_enumeration
integer_constant(3)
integer_constant(4)
integer_constant(5)
close_set_enumeration
type_check
evaluate
reset
difference
open_set_enumeration
integer_constant(3)
integer_constant(4)
integer_constant(5)
close_set_enumeration
open_set_enumeration
integer_constant(1)
integer_constant(2)
integer_constant(3)
close_set_enumeration
type_check
evaluate
