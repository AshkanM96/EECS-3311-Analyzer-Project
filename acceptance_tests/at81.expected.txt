  Expression currently specified: ?
  Report: Expression is initialized.
->union
  Expression currently specified: (? \/ nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({?} \/ nil)
  Report: OK.
->integer_constant(1)
  Expression currently specified: ({1, ?} \/ nil)
  Report: OK.
->integer_constant(2)
  Expression currently specified: ({1, 2, ?} \/ nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({1, 2, 3, ?} \/ nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({1, 2, 3} \/ ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({1, 2, 3} \/ {?})
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({1, 2, 3} \/ {3, ?})
  Report: OK.
->integer_constant(4)
  Expression currently specified: ({1, 2, 3} \/ {3, 4, ?})
  Report: OK.
->integer_constant(5)
  Expression currently specified: ({1, 2, 3} \/ {3, 4, 5, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({1, 2, 3} \/ {3, 4, 5})
  Report: OK.
->type_check
  Expression currently specified: ({1, 2, 3} \/ {3, 4, 5})
  Report: ({1, 2, 3} \/ {3, 4, 5}) is type-correct.
->evaluate
  Expression currently specified: ({1, 2, 3} \/ {3, 4, 5})
  Report: {1, 2, 3, 4, 5}
->reset
  Expression currently specified: ?
  Report: OK.
->intersect
  Expression currently specified: (? /\ nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({?} /\ nil)
  Report: OK.
->integer_constant(1)
  Expression currently specified: ({1, ?} /\ nil)
  Report: OK.
->integer_constant(2)
  Expression currently specified: ({1, 2, ?} /\ nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({1, 2, 3, ?} /\ nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({1, 2, 3} /\ ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({1, 2, 3} /\ {?})
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({1, 2, 3} /\ {3, ?})
  Report: OK.
->integer_constant(4)
  Expression currently specified: ({1, 2, 3} /\ {3, 4, ?})
  Report: OK.
->integer_constant(5)
  Expression currently specified: ({1, 2, 3} /\ {3, 4, 5, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({1, 2, 3} /\ {3, 4, 5})
  Report: OK.
->type_check
  Expression currently specified: ({1, 2, 3} /\ {3, 4, 5})
  Report: ({1, 2, 3} /\ {3, 4, 5}) is type-correct.
->evaluate
  Expression currently specified: ({1, 2, 3} /\ {3, 4, 5})
  Report: {3}
->reset
  Expression currently specified: ?
  Report: OK.
->difference
  Expression currently specified: (? \ nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({?} \ nil)
  Report: OK.
->integer_constant(1)
  Expression currently specified: ({1, ?} \ nil)
  Report: OK.
->integer_constant(2)
  Expression currently specified: ({1, 2, ?} \ nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({1, 2, 3, ?} \ nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({1, 2, 3} \ ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({1, 2, 3} \ {?})
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({1, 2, 3} \ {3, ?})
  Report: OK.
->integer_constant(4)
  Expression currently specified: ({1, 2, 3} \ {3, 4, ?})
  Report: OK.
->integer_constant(5)
  Expression currently specified: ({1, 2, 3} \ {3, 4, 5, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({1, 2, 3} \ {3, 4, 5})
  Report: OK.
->type_check
  Expression currently specified: ({1, 2, 3} \ {3, 4, 5})
  Report: ({1, 2, 3} \ {3, 4, 5}) is type-correct.
->evaluate
  Expression currently specified: ({1, 2, 3} \ {3, 4, 5})
  Report: {1, 2}
->reset
  Expression currently specified: ?
  Report: OK.
->difference
  Expression currently specified: (? \ nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({?} \ nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, ?} \ nil)
  Report: OK.
->integer_constant(4)
  Expression currently specified: ({3, 4, ?} \ nil)
  Report: OK.
->integer_constant(5)
  Expression currently specified: ({3, 4, 5, ?} \ nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, 4, 5} \ ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({3, 4, 5} \ {?})
  Report: OK.
->integer_constant(1)
  Expression currently specified: ({3, 4, 5} \ {1, ?})
  Report: OK.
->integer_constant(2)
  Expression currently specified: ({3, 4, 5} \ {1, 2, ?})
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, 4, 5} \ {1, 2, 3, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, 4, 5} \ {1, 2, 3})
  Report: OK.
->type_check
  Expression currently specified: ({3, 4, 5} \ {1, 2, 3})
  Report: ({3, 4, 5} \ {1, 2, 3}) is type-correct.
->evaluate
  Expression currently specified: ({3, 4, 5} \ {1, 2, 3})
  Report: {4, 5}
